sct-commons/src/main/java/org/lfenergy/compas/sct/commons/LdeviceService.java

```diff
@@ -41,6 +44,6 @@ public Optional<TLDevice> findLdevice(TIED tied, Predicate<TLDevice> ldevicePred
 
     public Optional<ActiveStatus> getLdeviceStatus(TLDevice tlDevice) {
         LnService lnService = new LnService();
```

Comment:
It seems the D of SOLID is not applied Dependency Injection
`LdeviceService` have only to define `LnService`.